plugins/woocommerce-blocks/assets/js/blocks/classic-template/README.md

```diff
@@ -4,12 +4,25 @@
 
 -   [Usage](#usage)
     -   [Props](#props)
+-   [Internals](#internals)
+-   [Extensibility](#extensibility)
+    -   [PHP actions and filters](#php-actions-and-filters)
+    -   [PHP template parts](#php-template-parts)
 
-The Classic Template block is a placeholder block for specific WooCommerce block templates which are rendered on the server-side when a block theme is active.
+The Classic Template block is a placeholder block for specific WooCommerce block templates which are rendered on the server side when a block theme is active. In the editor UI, it's presented with these names:
 
-By assigning a template identifier to the attribute prop, the block will render that specific template on the front-end, and a placeholder for said template in the Site Editor.
+- `Product (Classic)`,
+- `Product Attribute (Classic)`,
+- `Product Taxonomy (Classic)`,
+- `Product Tag (Classic)`,
+- `Product Search Results (Classic)`,
+- `Product Grid (Classic)`.
 
-It's worth noting that the placeholder in the Site Editor is merely an approximate representation of a template's front-end view.
+![Classic Template block in the Single Product template](https://github.com/woocommerce/woocommerce/assets/3616980/c996d4e8-8839-4542-b6a3-9f01627c482d)
```

Comment:
It would be great if we can move this image below following text:
> When the Classic Template block is clicked, a button appears to 'Transform into blocks'. Clicking it updates the template so it's composed of more granular blocks. That's what we commonly refer to as the 'blockified template'. The blockified template has several advantages over using the Classic Template block, like allowing users to hide some blocks from the template, changing their order or making more granular modifications.

Because the image is related to this text. It will help the reader to understand the context better. :slightly_smiling_face:

Reply:
This image is quite large, so maybe to keep the page clean, we could consider just linking to it, e.g.:

> When the Classic Template block is clicked, [a button appears to 'Transform into blocks'](https://github.com/woocommerce/woocommerce/assets/3616980/c996d4e8-8839-4542-b6a3-9f01627c482d). (…)

Reply:
Alternatively, we can set the image width:

```
<img src="https://github.com/woocommerce/woocommerce/assets/3616980/c996d4e8-8839-4542-b6a3-9f01627c482d" alt="Classic Template block in the Single Product template" width="400"/>
```

<img src="https://github.com/woocommerce/woocommerce/assets/3616980/c996d4e8-8839-4542-b6a3-9f01627c482d" alt="Classic Template block in the Single Product template" width="400"/>

Reply:
Ha, I was just about to suggest that, together with centering it! 😄 

```md
<p align="center">
  <img src="https://github.com/woocommerce/woocommerce/assets/3616980/c996d4e8-8839-4542-b6a3-9f01627c482d" alt="Classic Template block in the Single Product template" width="400"/>
</p>
```

<p align="center">
  <img src="https://github.com/woocommerce/woocommerce/assets/3616980/c996d4e8-8839-4542-b6a3-9f01627c482d" alt="Classic Template block in the Single Product template" width="400"/>
</p>

Reply:
Good feedback, folks. I updated it in the last commit!